roles/dd-nodepool/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 dd_nodepool_status_url:  http://locahost:8005/image-list
+dd_nodepool_images_url:  http://locahost:8005/dib-image-list
```

Comment:
This url gives a human readable table with the images in it.  It should hit /dib-image-list.json and that will give you something you can easily look at below.